external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -221,7 +224,22 @@ private boolean commit() {
     }
 
     private boolean poll() {
-        return !waitingToEmit() && numUncommittedOffsets < kafkaSpoutConfig.getMaxUncommittedOffsets();
+        final int maxUncommittedOffsets = kafkaSpoutConfig.getMaxUncommittedOffsets();
+        final boolean poll = !waitingToEmit() && numUncommittedOffsets < maxUncommittedOffsets;
+
+        if (!poll) {
+            final boolean isLoggable = logTimer.isExpiredResetOnTrue();
```

Comment:
Given that this is debug it won't print in normal op. If I turn on the debug logging my expectations is to see the logs immediately. If I don't see the messages than I will think that the debug log . 
Either way I am ok with it.